pages/linux/cat.md

```diff
@@ -0,0 +1,32 @@
+# cat
+
+> Print and concatenate files.
+> More information: <https://www.gnu.org/software/coreutils/cat>.
+
+- Print the contents of a file to the standard output:
+
+`cat {{path/to/file}}`
+
+- Concatenate several files into an output file:
+
+`cat {{path/to/file1 path/to/file2 ...}} > {{path/to/output_file}}`
+
+- Append several files to an output file:
+
+`cat {{path/to/file1 path/to/file2 ...}} >> {{path/to/output_file}}`
+
+- Copy the contents of a file into an output file without buffering:
+
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
+
+- Write `stdin` to a file:
+
+`cat - > {{path/to/file}}`
+
+- Number all output lines:
```

Comment:
```suggestion
- [n]umber all output lines:
```

The same issue as above.

Reply:
I haven't thought about mnemonics while doing these, good catch.